src/AnomalyDetectors/OneClassSVM.php

```diff
@@ -211,7 +218,13 @@ public function predictSample(array $sample) : int
             throw new RuntimeException('Estimator has not been trained.');
         }
 
-        return $this->model->predict($sample) !== 1.0 ? 0 : 1;
+        $sampleWithOffset = [];
+
+        foreach ($sample as $key => $value) {
+            $sampleWithOffset[$key + 1] = $value;
+        }
+
+        return $this->model->predict($sampleWithOffset) == 1 ? 0 : 1;
```

Comment:
I notice we are "inversing" the logic here i.e. 1 is now 0, 0 is now 1. Is that intentional?

Reply:
Yes, in fact in the one class mode of libsvm, the "normal" samples are to be labelled with the 1 class. And the anomalies, are to be labelled with -1. That's why !